### ISSUE TITLE
Invalid test: LanguageFeatures/Spread-collections/Ambiguity_A01_t03

### ISSUE THREAD

**Issue by bwilkerson** [dart-lang/co19#269]
This produces the following errors, all of which appear to be valid:
```
ERROR|STATIC_WARNING|UNDEFINED_IDENTIFIER|/b/s/w/ir/cache/builder/sdk/tests/co19_2/src/LanguageFeatures/Spread-collections/Ambiguity_A01_t03.dart|69|25|3|Undefined name 'map'.
ERROR|STATIC_WARNING|UNDEFINED_IDENTIFIER|/b/s/w/ir/cache/builder/sdk/tests/co19_2/src/LanguageFeatures/Spread-collections/Ambiguity_A01_t03.dart|67|19|3|Undefined name 'map'.
ERROR|SYNTACTIC_ERROR|EXPECTED_TOKEN|/b/s/w/ir/cache/builder/sdk/tests/co19_2/src/LanguageFeatures/Spread-collections/Ambiguity_A01_t03.dart|69|28|1|Expected to find ';'.
ERROR|STATIC_WARNING|UNDEFINED_IDENTIFIER|/b/s/w/ir/cache/builder/sdk/tests/co19_2/src/LanguageFeatures/Spread-collections/Ambiguity_A01_t03.dart|70|49|3|Undefined name 'map'.
ERROR|STATIC_WARNING|UNDEFINED_IDENTIFIER|/b/s/w/ir/cache/builder/sdk/tests/co19_2/src/LanguageFeatures/Spread-collections/Ambiguity_A01_t03.dart|71|51|3|Undefined name 'map'.
```

**Comment by iarkh** [comment]
This is already fixed on March, 1 - please check revision 86b599b739111e4402cd23e83a37d80c8c51d241

**Comment by danrubel** [comment]
In that revision, it appears that there is a missing semicolon on line 69 of LanguageFeatures/Spread-collections/Ambiguity_A01_t03.dart
```
  Set res5 = {...i, ...?map}
```
